generator/testing/error_collectors.h

```diff
@@ -26,21 +27,21 @@ class ErrorCollector : public google::protobuf::DescriptorPool::ErrorCollector {
  public:
   ~ErrorCollector() override = default;
 
-  void AddError(std::string const& filename, std::string const& element_name,
-                google::protobuf::Message const*, ErrorLocation,
-                std::string const& error_message) override;
+  void RecordError(absl::string_view filename, absl::string_view element_name,
+                   google::protobuf::Message const*, ErrorLocation,
+                   absl::string_view error_message) override;
```

Comment:
The one difference from my version is that I also changed `error_message` to just `message` to match the `DescriptorPool::ErrorCollector` declaration.

Reply:
Done.

Reply:
For the record, I also verified that the change also built cleanly with v26.0 (unsurprisingly).